apps/website/app/ui/sections/ColorsSection.tsx

```diff
@@ -0,0 +1,65 @@
+import { Card, Section } from "@/components";
+import resolveConfig from "tailwindcss/resolveConfig";
+import tailwindConfig from "@/tailwind.config";
+
+const colorsKeysBanList = [
+  "transparent",
+  "inherit",
+  "elevation-shadow", // todo: fix appearing 20 versions when is just a color
+  "transparent_inverse", // todo: fix appearing 20 versions when is just a color
```

Comment:
The underscore should be a dash, right?
Also, why do we have a ban list?

```suggestion
  "transparent-inverse", 
```